pkgs/test_core/lib/src/runner.dart

```diff
@@ -298,6 +299,30 @@ class Runner {
             return false;
           }
 
+          // Skip tests that don't start on `line` or `col` if specified.
+          var line = suite.config.line;
+          var col = suite.config.col;
+          var trace = test.trace;
+          if (trace == null && (line != null || col != null)) {
```

Comment:
When would we be missing a trace? Should we consider it an error to try to run tests with a line and column when we have no trace available?

Reply:
I am not sure if we get load tests and things through this api. I will try making it an error and see what happens.